Jellyfin.Drawing.Skia/StripCollageBuilder.cs

```diff
@@ -78,20 +78,15 @@ private SKBitmap BuildThumbCollageBitmap(string[] paths, int width, int height)
                 canvas.Clear(SKColors.Black);
 
                 // determine sizes for each image that will composited into the final image
-                var iSlice = Convert.ToInt32(width * 0.23475);
-                int iTrans = Convert.ToInt32(height * .25);
-                int iHeight = Convert.ToInt32(height * .70);
-                var horizontalImagePadding = Convert.ToInt32(width * 0.0125);
-                var verticalSpacing = Convert.ToInt32(height * 0.01111111111111111111111111111111);
+                var iSlice = Convert.ToInt32(width * 0.33);
+                int iTrans = Convert.ToInt32(height * 0.25);
+                int iHeight = Convert.ToInt32(height * 1.00);
                 int imageIndex = 0;
-
-                for (int i = 0; i < 4; i++)
+                for (int i = 0; i < 3; i++)
```

Comment:
Maybe the magic number 3 should be put in a const variable that better describes what it is